Addition of LR1110 driver

kvkc97 opened the pull request.

paul-szczepanek-arm commented:
Hi, this is showing 137,536 lines of changes and a conflicted file. If you're having trouble rebasing you can simply make a fresh checkout and re-apply your changes. Otherwise it's hard for others to see what you're doing.

jeromecoutant commented:
yes, remove:
- qspi addation in cpp file
- TARGET_MY_TARGET directory
- cmake_buiild directory

kvkc97 commented:
Thanks for the suggestion. I have removed the mentioned files and updated the code. Also some changes was made in the mbed-os folder which could not be added to this repository. Therefore, I have created a separate repository for mbed-os library and made the changes. Attached is the link for the same.
https://github.com/kvkc97/mbed-os
The only change in mbed-os library is the addition of the COMPONENT_LR1110 folder at the location mbed-os\connectivity\drivers\lora

0xc0170 commented:
@kvkc97 would you be able to rebase , there should be linear history (no merge commits) as indicated in our guidelines

kvkc97 commented:
I have created a new repo for he code with the latest mbed-os-example-lorawan code.
https://github.com/kvkc97/mbed-os-example-lorawan-lr1110
Since I could not attach the mbed-os library and the changes, i is attached in the below link.
https://github.com/kvkc97/mbed-os
